src/validations/access-checker/wearables.ts

```diff
@@ -425,40 +392,61 @@ export const wearables: Validation = {
       return validMerkleProofedEntity
     }
 
+    const parseUrnNoFail = async (urn: string): Promise<DecentralandAssetIdentifier | null> => {
+      try {
+        return await parseUrn(urn)
+      } catch (err) {
+        if (err instanceof Error) {
+          logs?.error(err)
+        } else {
+          logs?.error(`Error parsing urn: ${err}`)
+        }
+      }
+      return null
+    }
+
     const { pointers } = deployment.entity
     const ethAddress = externalCalls.ownerAddress(deployment.auditInfo)
 
-    const resolvedPointers: SupportedAssets[] = []
-    // deduplicate pointer resolution
-    for (const pointer of pointers) {
-      const parsed = await parseUrnNoFail(pointer)
-      if (!parsed)
-        return validationFailed(
-          `Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointer})`
-        )
-
-      if (!alreadySeen(resolvedPointers, parsed)) resolvedPointers.push(parsed)
+    if (pointers.length == 0) {
+      return validationFailed('Wearable should specify a pointer')
     }
 
-    if (resolvedPointers.length > 1)
-      return validationFailed(`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`)
-
-    const parsed = resolvedPointers[0]
+    const wearablePointer: DecentralandAssetIdentifier | null = await parseUrnNoFail(pointers[0])
+    if (wearablePointer === null) {
+      return validationFailed(`Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointers[0]})`)
+    }
 
-    if (!validUrnTypes.includes(parsed.type)) {
-      return validationFailed(`Could not resolve the contractAddress of the urn ${parsed}`)
+    const entityTimestampIsAfterADR45 = deployment.entity.timestamp >= ADR_45_TIMESTAMP
+    if (entityTimestampIsAfterADR45) {
```

Comment:
is this really needed? Is it a problem if multiple pointers resolve to the same URN?

Reply:
Technically, there's no much problem. But could lead to complicated designs.
We're working in an endpoint to ask urn by prefix, a feature requested by dApps. 
In that context, if a wearable was deployed using only one pointer, which version of the urn should use the client? One requests per possible urn?
If a wearable was deployed using two pointers, the prefix requests would work for two version of the urns, but how many urns that resolve to the same thing exist?
For that reason, we think it would be nice to standardize how wearables are deployed.
Also, AgusA is working in a RFC for the filter by urn prefix, and we could see more details there.

Reply:
currently, we can take a look at the deployed wearables. many have many URNs with common names for contracts (mostly, if not entirely, L1) and then the contract address, that's the flag I wanted to raise. 

I recommend you to finish up this decision with @nachomazzara and Dapps team

Reply:
Yes, this decision came from a discussion with @nachomazzara but will double-check.

Reply:
Could you provide an example of what does it mean one pointer per wearable?

Reply:
@nachomazzara 

This entity has two pointers:
https://peer-testing-2.decentraland.org/content/deployments/?pointer=urn:decentraland:ethereum:collections-v1:exclusive_masks:classic_mask

This one has only one:
https://peer-testing-2.decentraland.org/content/deployments/?pointer=urn:decentraland:matic:collections-v2:0xf1ec64da82847bc0511c23d16794bc7e1683731f:2

Reply:
I'll write an ADR to address this discussion

Reply:
Here is the PR of the ADR that address this subject: https://github.com/decentraland/adr/pull/54